src/main/java/frc/robot/AutoRoutes.java

```diff
@@ -11,116 +11,232 @@
+                        30);
+        private static final Rotation2d DEGREES_45  = Rotation2d.fromDegrees(
+                        45);
+        private static final Rotation2d DEGREES_60  = Rotation2d.fromDegrees(
+                        60);
+        private static final Rotation2d DEGREES_90  = Rotation2d.fromDegrees(
+                        90);
+        private static final Rotation2d DEGREES_120 = Rotation2d.fromDegrees(
+                        120);
+        private static final Rotation2d DEGREES_135 = Rotation2d.fromDegrees(
+                        135);
+        private static final Rotation2d DEGREES_150 = Rotation2d.fromDegrees(
+                        150);
+        private static final Rotation2d DEGREES_180 = Rotation2d.fromDegrees(
+                        180);
+
+        // Define common positions
+        public static final Pose2d START_POINT_3 = new Pose2d(2.2, 4.7,
+                        DEGREES_180);
+        public static final Pose2d START_POINT_2 = new Pose2d(2.2, 2.75,
+                        DEGREES_180);
+        public static final Pose2d START_POINT_1 = new Pose2d(2.2, 0.75,
+                        DEGREES_180);
+
+        public static final Pose2d GAME_PIECE_4 = new Pose2d(6.8, 4.6,
+                        DEGREES_0);
+        public static final Pose2d GAME_PIECE_3 = new Pose2d(6.8, 3.35,
+                        DEGREES_0);
+        public static final Pose2d GAME_PIECE_2 = new Pose2d(6.8, 2.15,
+                        DEGREES_0);
+        public static final Pose2d GAME_PIECE_1 = new Pose2d(6.8, 0.9,
+                        DEGREES_0);
+
+        // Define charge station points, counter-clockwise, starting from bottom
+        // left of blue
+        public static final Pose2d CS_EDGE_1 = new Pose2d(2.9, 1.5, DEGREES_0);
+        public static final Pose2d CS_EDGE_2 = new Pose2d(4.8, 4.0, DEGREES_0);
+        public static final Pose2d CS_EDGE_3 = new Pose2d(4.8, 1.5, DEGREES_0);
+        public static final Pose2d CS_EDGE_4 = new Pose2d(2.9, 4.0, DEGREES_0);
+        public static final Pose2d CS_CENTER = new Pose2d(3.9, 2.75, DEGREES_0);
+
+        /*
+         * Define specific routes Eventually maybe add mid points to have not a
+         * sharp rotation We do not want to have each one rotating
+         */
+
+        // Start 1 Piece 1 Drive to piece
+        private static final Pose2d S1_P1_A = new Pose2d(1.49, 1.07, DEGREES_0);
+        private static final Pose2d S1_P1_B = new Pose2d(2.94, 0.92, DEGREES_0);
+        private static final Pose2d S1_P1_C = new Pose2d(4.83, 0.92, DEGREES_0);
+        private static final Pose2d S1_P1_D = new Pose2d(6.5, 0.92,
+                        DEGREES_180);
+        private static final Pose2d S1_P1_E = new Pose2d(6.82, 0.92,
+                        DEGREES_180);
+
+        public static final Pose2d[] START_1_TO_PIECE_1 = { S1_P1_A, S1_P1_B,
+                        S1_P1_C, S1_P1_D, S1_P1_E };
```

Comment:
These arrays must also include the start and end position, or else the robot will be very confused at the start!

Reply:
This applies to every array in this file

Reply:
We removed those as it looked like it in your code it was referencing these positions. "public static final Pose2d START_POINT_1 = new Pose2d(2.2, 4.7, DEGREES_180);
    public static final Pose2d START_POINT_3 = new Pose2d(2.2, 4.7, DEGREES_180);
    public static final Pose2d START_POINT_2 = new Pose2d(2.2, 2.75, DEGREES_180);
    public static final Pose2d START_POINT_3 = new Pose2d(2.2, 0.75, DEGREES_180);" 
As S1_P1_A is our starting position we kept it out